### ISSUE TITLE
Training model

### ISSUE THREAD

**Issue by Luyang1125** [Zheng222/IDN-Caffe#2]
Hi,

I have a question about training an IDN model myself. I saw you mentioned in the paper that you use l2 first and then use l1 to fine tune it. So can I understand you use l2 as the loss for the first 10^5 iterations, then l1 as the loss for the rest 5*10^5 iteration? Not sure if I understand correctly. Can you provide more information on how you trained the model?

**Comment by Zheng222** [comment]
Hello, you can first train the model with original solver.prototxt and then change the training patches (e.g. 29x29 --> 39x39 for 2x) for fine-tuning the model about 20k iterations. @Luyang1125
